src/main/java/uk/gov/hmcts/reform/bsp/common/model/shared/in/ExceptionRecord.java

```diff
@@ -1,27 +1,35 @@
 package uk.gov.hmcts.reform.bsp.common.model.shared.in;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 
 import java.util.List;
 
 @Getter
 @Builder
+@AllArgsConstructor
```

Comment:
whats the reason for need both of these constructors? https://projectlombok.org/features/constructor

Reply:
Hi @shaunganley. Jackson needs a no-param constructor and I think we were already using a constructor with all parameters in the tests.